src/Shared/Model/Enums/PackageDoesNotExistReason.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+
+namespace Microsoft.CST.OpenSource.Model.Enums;
+
+/// <summary>
+/// Enum for information about when a package no longer exists on a package manager.
+/// </summary>
+public enum PackageDoesNotExistReason
+{
+    /// <summary>
+    /// When the version was unpublished from the repository/registry by someone.
+    /// </summary>
+    VersionUnpublished,
+    
+    /// <summary>
+    /// When the package was unpublished from the repository/registry by someone.
+    /// </summary>
+    PackageUnpublished,
+
+    /// <summary>
+    /// When the package/version was removed from the repository/registry by the package manager's administrators due to a security risk.
+    /// </summary>
+    RemovedForSecurity,
```

Comment:
`RemovedForSecurity` is pretty vague. Can we be more specific?

Maybe something like `ConsideredMalicious`? That would emphasize 'this looks like malware' as opposed to just has a known vulnerability/CVE.

Reply:
This is just trying to convey when the registry itself pulls a package/version from the registry. I'd imagine that the only use case there is for "Malicious" purposes, so I think `ConsideredMalicious` makes sense, but if anyone else has another suggestion let me know!

Reply:
I think both `RemovedForSecurity` and `ConsideredMalicious` convey a sense of confidence in the rationale for removal we don’t truly have. I recommend `RemovedByRepository`